pipelines/pr.yaml

```
@@ -38,4 +38,38 @@ jobs:
       buildUWPArm: false
       buildUWPDotNet: false
       UnityVersion: $(Unity2018Version)
+  
+- job: Unity2019Validation
+  timeoutInMinutes: 90
+  pool:
+    name: On-Prem Unity
+    demands:
+    - ${{ variables.Unity2019Version }}
+    - COG-UnityCache-WUS2-01
+    - SDK_18362 -equals TRUE
+    - VS2019 -equals TRUE
+  steps:
+  - template: templates/common.yml
+    parameters:
+      buildUWPX86: false
+      buildUWPArm: false
+      buildUWPDotNet: false
+      UnityVersion: $(Unity2019Version)
+
+  - job: Unity2020Validation
```

I think there's a slight whitespace discrepancy here

```suggestion
- job: Unity2020Validation
```